DIGNDB.App.SmitteStop.API/appsettings.json

```diff
@@ -73,6 +73,59 @@
     "TransmissionRiskScores": [ 1, 1, 8, 8, 8, 8, 1, 1 ],
     "TransmissionRiskWeight": 50
   },
+  "DailySummaryConfiguration": {
+    "AttenuationThresholds": [ 50, 70, 90 ],
+    "AttenuationWeights": {
+      "Immediate": 2.5,
+      "Near": 2.5,
+      "Medium": 2.5,
+      "Other": 2.5
+    },
+    "DaysSinceLastExposureThreshold": 0,
+    "DaysSinceOnsetInfectiousness": {
+      "-14": 1,
+      "-13": 1,
+      "-12": 1,
+      "-11": 1,
+      "-10": 1,
+      "-9": 1,
+      "-8": 1,
+      "-7": 1,
+      "-6": 1,
+      "-5": 1,
+      "-4": 1,
+      "-3": 1,
+      "-2": 2,
+      "-1": 2,
+      "0": 2,
+      "1": 2,
+      "2": 2,
+      "3": 2,
+      "4": 2,
+      "5": 2,
+      "6": 2,
+      "7": 2,
+      "8": 2,
+      "9": 1,
+      "10": 1,
+      "11": 1,
+      "12": 1,
+      "13": 1,
+      "14": 1
```

Comment:
For days from -14 to -6 and from 10 to 14 set 0,
For days from -5 to -4 and from 5 to 9 set 1,
For days from -3 to 4 set 2